pubnub-gson/pubnub-gson-api/src/main/java/com/pubnub/api/java/models/consumer/objects_api/channel/PNChannelMetadata.java

```diff
@@ -43,14 +45,14 @@ public PNChannelMetadata setCustom(Object custom) {
     public static PNChannelMetadata from(@NotNull com.pubnub.api.models.consumer.objects.channel.PNChannelMetadata data) {
         PNChannelMetadata newData = new PNChannelMetadata(
                 data.getId(),
-                data.getName() != null ? data.getName().getValue() : null,
-                data.getDescription() != null ? data.getDescription().getValue() : null
+                data.getName(),
+                data.getDescription()
         );
-        newData.setETag(data.getETag() != null ? data.getETag().getValue() : null);
-        newData.setType(data.getType() != null ? data.getType().getValue() : null);
-        newData.setStatus(data.getStatus() != null ? data.getStatus().getValue() : null);
-        newData.setCustom(data.getCustom() != null ? data.getCustom().getValue() : null);
-        newData.setUpdated(data.getUpdated() != null ? data.getUpdated().getValue() : null);
+        newData.setETag(data.getETag());
+        newData.setType(data.getType());
+        newData.setStatus(data.getStatus());
+        newData.setCustom(data.getCustom() != null ? PatchValue.of(data.getCustom().getValue()) : null);
```

Comment:
Why Custom is handled differently than other properties?

Reply:
because in Java for some reason we have Object (so now PatchValue<Object>) and not Map, and PatchValue<Map> is not a PatchValue<Object>... maybe I'll just change it to Map for consistency instead...

Reply:
done